projects/js-packages/licensing/components/activation-screen-success-info/product-details/index.js

```diff
@@ -92,7 +92,7 @@ const JetpackProductDetails = props => {
 			),
 		},
 		jetpack_videopress: {
-			title: __( 'Your Jetpack Videopress is active!', 'jetpack' ),
+			title: __( 'Jetpack Videopress is active!', 'jetpack' ),
```

Comment:
It appears this was already a pre-existing issue, but I think the product "Videopress" should have a capitol "P" in VideoPress, right?  Since this PR is already created, I think this would be a good opportunity to go ahead and fix this too. 🙂

Reply:
Good catch, thanks. Yes, I definitely think it worth updating.

Reply:
Pushed the changes and updated the screenshot in PR description.